integration-tests/tests/create-application-from-public-git-source.spec.ts

```diff
@@ -86,15 +81,17 @@ describe('Create Component from Public Git Source', { tags: ['@PR-check', '@publ
       applicationDetailPage.checkReplica(replicaCount);
     });
 
-    // Skipping due to : https://issues.redhat.com/browse/HAC-3184
-    // it('Change Resources Value', () => {
-    //   applicationDetailPage.openComponentSettings(componentPage.componentName);
-    //   componentPage.setRam(2, MemoryUnit.gigabyte);
-    //   componentPage.setCpuByButton(cpuCount, cpuUnit);
-    //   componentPage.saveChanges();
-    //   Applications.goToComponentsTab();
-    //   applicationDetailPage.expandDetails(componentPage.componentName);
-    //   applicationDetailPage.checkCpuAndMemory(cpuCount, CPUUnit.millicore, 2, MemoryUnit.gigabyte);
-    // });
+    it('Change Resources Value', () => {
```

Comment:
What about having two last tests in a different suite then `Creating a Quarkus Component`, as the component is already created?

Reply:
I didn't understood, though there is `Creating a Quarkus Component` which I will update to `Creating a Nodejs Component` as I have update the repo from Quarkus to nodejs

Reply:
I mean the component is created on line 73. So, the changes of resources done in the following two tests are changing the resources of an existing app. So I propose doing something like that:

line 78:
```
describe('Change resources for the existing app', () => {
   it('Check Resources Value', () => {
       ...
   });

   it('Change Resources Value', () => {
      ...
   });
});
```